web/client/components/data/identify/Identify.jsx

```diff
@@ -118,9 +120,37 @@ class Identify extends React.Component {
         closeGlyph: "1-close",
         className: "square-button",
         allowMultiselection: false,
-        currentLocale: 'en-US'
+        currentLocale: 'en-US',
+        fullscreen: false
     };
 
+    state = {
+        fullClass: ''
+    };
+
+    componentWillMount() {
+        if (this.props.fullscreen) {
+            /* compatibility with Safari*/
+            const touchStream$ = new Rx.Subject();
+            touchStream$
```

Comment:
I found good recompose functions to wrap a component and control properties using streams. It works good also because takes care for you of stream close and so on. It's ok, but take a look for them in the future, could be simpler to write and maintain, without using internal state

Reply:
if we use a button with single click instead of double click/touch, this stream is no more needed.

button or double click/touch ? @offtherailz @tdipisa